zhaquirks/tuya/ts0601_illuminance.py

```diff
@@ -0,0 +1,115 @@
+"""Tuya HOME-LUX illuminance sensor."""
+
+import math
+from typing import Dict
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, GreenPowerProxy, Groups, Ota, Scenes, Time
+from zigpy.zcl.clusters.measurement import IlluminanceMeasurement
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.tuya import DPToAttributeMapping, TuyaLocalCluster, TuyaNewManufCluster
+
+TUYA_BRIGHTNESS_LEVEL_ATTR = 0x01  # 0-2 "Low, Medium, High"
+TUYA_ILLUMINANCE_ATTR = 0x02  # [0, 0, 3, 232] illuminance
+
+TUYA_BRIGHTNESS_LEVEL_STR = ["Low", "Medium", "High"]
```

Comment:
What about if we use:
```python
class BrightnessLevel(t.enum8):
    """Brightness level enum."""

    LOW = 0x00
    MEDIUM = 0x01
    HIGH = 0x02
```

That way you can define the attribute as:
```python
            0xFF00: ("brightness_level", BrightnessLevel, True),
```

And convert as:
```python
    dp_to_attribute: Dict[int, DPToAttributeMapping] = {
        TUYA_BRIGHTNESS_LEVEL_ATTR: DPToAttributeMapping(
            TuyaIlluminanceMeasurement.ep_attribute,
            "brightness_level",
            lambda x: BrightnessLevel(x),
        ),
```